main.go

```diff
@@ -537,12 +526,26 @@ func main() {
 	// 	log.Fatal(err)
 	// }
 
-	if err := runner.Run(ctx, conn, lastRanMigrationID); err != nil {
-		log.Fatal(err)
-	}
-
+	var org string = "default"
 	app := &cli.App{
 		Name: "store",
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:        "org",
+				Value:       "default",
+				Usage:       "org to connect to",
+				Destination: &org,
+			},
+		},
+		Action: func(cCtx *cli.Context) error {
```

Comment:
It might be better to use `BeforeFunc` instead of Action (https://pkg.go.dev/github.com/urfave/cli/v2#App) since I think Action only execute if no subcommands are specified. 
```suggestion
		Before: func(cCtx *cli.Context) error {
```